tests/acceptance/gist-test.js

```diff
@@ -55,7 +55,7 @@ test('deleting a gist loaded in two columns', function(assert) {
     });
 
     // TODO: Replace brittle for loop test code with "while there are files left..."
-    for (var i = 0; i < 2; ++i) {
+    for (var i = 0; i < 1; ++i) {
```

Comment:
I'm not sure what was going on here, but it appears this didn't run on `head` either. It would delete the `twiddle.json` and cause a number of cascading test failures. Is `twiddle.json` supposed to be impossible to delete?

Reply:
yes, it is supposed to be impossible to delete, in that doing so recreates it.